SuperCalculator/src/supercalculator/Tan.java

```diff
@@ -0,0 +1,131 @@
+
+package supercalculator;
+
+import java.util.Scanner;
+/*
+ * Tangent class
```

Comment:
I think the Oracle code convention determines this kind of comment to be before the package and import declarations as per sections 3.1.1 and 3.1.2. Must check these in all files changed to make sure that's correct (unless of course I misread the convention).

Reply:
@Leandro190 as per section 3.1.1 this is related to the class. Also after using the "Checkstyle" tool with oracle code quality. It should be before the class